egg/zoo/compo_vs_generalization/callbacks.py

```diff
@@ -0,0 +1,47 @@
+import egg.core as core
+from egg.core.interaction import Interaction
+from egg.core.util import reset_optimizer_state
+from egg.zoo.compo_vs_generalization.curriculum_games import CurriculumGameWrapper
+
+import torch
+import numpy as np
+
+
+class CurriculumManager(core.Callback):
```

Comment:
The name is a bit too generic for what you are doing (you could manage the curriculum in a different way).

How about making the name more explicit and adding a quick docstring to explain the strategy as well as the arguments?